lib/images/icons/badge-team.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="12" height="12" xmlns:v="https://vecta.io/nano"><defs><filter id="A"><feColorMatrix in="SourceGraphic" values="0 0 0 0 0.435294 0 0 0 0 0.447059 0 0 0 0 0.521569 0 0 0 1.000000 0"/></filter><path d="M10.839.977c1.254 1.254 1.301 3.259.139 4.569l-.139.148-5.145 5.145a3.32 3.32 0 0 1-2.074.965l-.189.011h0-.19c-.822-.023-1.636-.348-2.263-.975C-.277 9.584-.324 7.58.837 6.27l.139-.148L6.122.977c1.302-1.302 3.414-1.302 4.717 0zm-3.773.943L4.842 4.143l2.83 2.83L9.895 4.75a2 2 0 1 0-2.83-2.83z" id="B"/></defs><g transform="translate(-8 -4)" filter="url(#A)" fill="none" fill-rule="evenodd"><g transform="translate(8 4)"><use fill="#f91bd6" xlink:href="#B"/></g></g></svg>
```

Comment:
`xmlns:v="https://vecta.io/nano"`?

Reply:
Imo `fill` should be set to `fill="currentColor"` and we should apply color in CSS